.github/ISSUE_TEMPLATE/feature_request.yaml

```
@@ -0,0 +1,38 @@
+name: "🚀 Feature request"
+description: Suggest a new feature
+labels: kind:feature-request
+body:
+  - type: dropdown
+    id: audience
+    attributes:
+      label: Who would benefit from this feature?
+      options: ["End users", "Software developers"]
+    validations:
+      required: true
+
+  - type: textarea
+    id: abilities
+    attributes:
+      label: What would this feature bring?
+      value: |
+        - users would be able to ...
+        - ... should be visible on the map
+        - when using ..., users should be able to ...
+    validations:
+      required: true
+
+  - type: textarea
+    id: value
+    attributes:
+      label: Why do you think this feature would be valuable?
+      value: This feature enables users to ...
```

```suggestion
      value:
          - This feature is necessary for ...
```

Otherwise i'm affraid this looks too much like the description

---

I used the word "necessary" as well in a previous iteration, and another reviewer (@Tguisnet, @flomonster or @Khoyo, I can't say for sure) mentioned that not all feature requests don't have to be strictly necessary for anything: a great improvement to something that's already there is a feature, but is it necessary?

I suggest this instead: `This feature enables ...`

---

Ok i guess necessary isn't the right word, but i just want to make sure that this field remains different from the description. I'm affraid `This feature enables ...` ends up in a redudant description of the actions the user will be able to do.

Another suggestion :

In the description section :
```
label: What is this feature about ?
```
instead of `label: What would this feature bring?`

And in the next section :    
```
value: This feature enables ...
This feature is important because....
This feature will make ... easier
```

---

Ok, I see what you mean. I settled on this:
```yaml
      label: What is this feature about?
      value: |
        - adding a new API for accessing ...
        - making ... visible on the map
	- adding a spinning icon when loading ...

      label: Why do you think this feature would be valuable?
      value: |
        This feature matters because it:
        - enables ...
        - will make ... easier
```

---

Fine by me !